### PR TITLE
Re-adding lost SSL content

### PR DESCRIPTION
**Pull request by Loquacity**
Recreates https://github.com/SUSE/doc-susemanager/pull/853/files which got lost in the move to the new repo.

### CODE REVIEW

**Comment by mcalmer** [comment]
Something with the self signed chapter seems to be incorrect. But i need to do a deeper check

File: modules/administration/pages/ssl-certs.adoc [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+{productname} uses SSL certificates to ensure that clients are registered to the correct server.
+
+When you create an SSL certificate for your server, the certificate includes the host name of the server it is associated with.
+The certificate is then deployed on the server, and when clients attempt to connect, they check the certificate details.
+If the certificate contains the name of the server they are attempting to connect to, the connection will be successful.
+The client will also check if the certificate has been signed by a trusted certificate authority.
+
+All certificates must be signed by a certificate authority (CA), in order for them to be considered valid, and for clients to be able to successfully match against them.
+When an organization signs its own certificate, the certificate is considered self-signed.
+A self-signed certificate is easy to set up, and does not cost any money, but they are considered less secure.
```
Comment (keichwa) [inline modules/administration/pages/ssl-certs.adoc]: "easy" is one of the forbidden words ;)  [...] straight forward to set up?

File: modules/administration/pages/ssl-certs.adoc [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+For additional security, you can arrange a third party CA to sign your certificates.
+Third party CAs perform checks to ensure that the information contained in the certificate is correct.
+They will usually charge an annual fee for this service.
+Using a third party CA makes certificates harder to spoof, and will provide additional protection for your installation.
+
+By default, {productname} uses a self-signed certificate.
+If you have certificates signed by a third party CA, you can import them to your {productname} installation.
+
+For more on self-signed certificates, see xref:administration:ssl-certs-selfsigned.adoc[].
```
Comment (keichwa) [inline modules/administration/pages/ssl-certs.adoc]: It's a list.  Let's use list decoration:
```suggestion
* For more on self-signed certificates, see xref:administration:ssl-certs-selfsigned.adoc[].
```

File: modules/administration/pages/ssl-certs.adoc [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
```
Comment (keichwa) [inline modules/administration/pages/ssl-certs.adoc]:
```suggestion
```

File: modules/administration/pages/ssl-certs.adoc [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+For more on imported certificates, see xref:administration:ssl-certs-imported.adoc[].
```
Comment (keichwa) [inline modules/administration/pages/ssl-certs.adoc]:
```suggestion
* For more on imported certificates, see xref:administration:ssl-certs-imported.adoc[].
```

File: modules/administration/pages/ssl-certs-selfsigned.adoc [hunk 5]
```diff
@@ -1,9 +1,15 @@
-[[custom-ssl]]
-= Custom SSL Certificates
+[[ssl-certs-selfsigned]]
+= Self-Signed SSL Certificates
 
-You can use custom certificates with {productname} and {productname} Proxy.
+By default, {productname} uses a self-signed certificate.
+In this case, the certificate is created and signed by {productname}.
+This method does not use an independent certificate authority to guarantee that the details of the certificate are correct.
+Third party CAs perform checks to ensure that the information contained in the certificate is correct.
```
Comment (mcalmer) [inline modules/administration/pages/ssl-certs-selfsigned.adoc]: We could refer to "Import SSL Certificates" chapter here. But this is optional.

File: modules/administration/pages/ssl-certs-selfsigned.adoc [hunk 6]
```diff
@@ -14,66 +20,12 @@ Before you begin, ensure you have:
 Your key and certificate files must be in PEM format.
```
Comment (mcalmer) [inline modules/administration/pages/ssl-certs-selfsigned.adoc]: This section is not correct in this chapter. We re-generate all of it, so you do not need them beforehand.
This is only important in the "Import" case

File: modules/administration/pages/ssl-certs.adoc (hunk 1 above)
Comment (Loquacity) [reply modules/administration/pages/ssl-certs.adoc]: yep, straight-forward works

File: modules/administration/pages/ssl-certs-selfsigned.adoc (hunk 6 above)
Comment (mcalmer) [reply modules/administration/pages/ssl-certs-selfsigned.adoc]: Line 14 - 20 can be removed

File: modules/administration/pages/ssl-certs-imported.adoc [hunk 7]
```diff
@@ -0,0 +1,129 @@
+[[ssl-certs-imported]]
+= Imported SSL Certificates
```
Comment (mcalmer) [inline modules/administration/pages/ssl-certs-imported.adoc]: Not sure if "Imported SSL Certificates" is correct. 

We want to describe how to "import" SSL Certificates. But the certificate itself is not "imported".
Maybe we talk about just (How to) "Import SSL Certificates" or "Import 3rd patry (signed) SSL Certificates".

File: modules/administration/pages/ssl-certs-selfsigned.adoc [hunk 8]
```diff
@@ -1,79 +1,25 @@
...
+This method does not use an independent certificate authority to guarantee that the details of the certificate are correct.
+Third party CAs perform checks to ensure that the information contained in the certificate is correct.
+For more on third party CAs, see xref:administration:ssl-certs-imported.adoc[].
+
+This section covers how to re-create your self-signed certificates on an existing installation.
+It also covers how to create new self-signed certificates and authenticate your existing clients to the new certificate, using an intermediate certificate.
+Intermediate certificates merge the intermediate and root CA certificates into one file.
+Ensure that the intermediate certificate comes first in the combined file.
 
 Your key and certificate files must be in PEM format.
```
Comment (mcalmer) [inline modules/administration/pages/ssl-certs-selfsigned.adoc]: "Your key and certificate files must be in PEM format." can be removed. We generate it in the correct format already.

**Pull request merged**